utils/symbol.c

```diff
@@ -1771,7 +1771,7 @@ TEST_CASE(symbol_load_module) {
 
 	save_module_symbol(&stab, symfile, 0x400000);
```

Comment:
Then I think you should rename `save_module_symbol_file()` as well. :)

Reply:
Done it. Thanks.